ansible/osp_cleanup.yaml

```diff
@@ -12,14 +12,21 @@
         PATH: "{{ oc_env_path }}"
         KUBECONFIG: "{{ kubeconfig }}"
       block:
-        - name: switch to openstack project/namespace
-          shell: >
-            oc project openstack
 
         - name: delete openstack.org CRD's
           shell: >
-            oc get crds | grep openstack.org | cut -f1 -d ' ' | xargs -t oc delete crds --cascade
+            oc get crds | grep openstack.org | cut -f1 -d ' ' | xargs -r -t oc delete crds --cascade
```

Comment:
I think we should change the order, 
* first scale down any existing osp worker nodes
* then remove the crds

otherwise draining of any existing compute worker node will fail as the openstack api might be already not reachable.

What I usually do is:
* delete any compute-node-operator CR to get osp worker nodes removed
* make olm_cleanup
* oc delete project openstack

Reply:
ok, updated.